FlashLFQ/FlashLFQ.csproj

```diff
@@ -5,10 +5,14 @@
   </PropertyGroup>
   
   <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
-    <DefineConstants>NETFRAMEWORK</DefineConstants>
+    <DefineConstants>DEBUG;NETFRAMEWORK;NET461</DefineConstants>
```

Comment:
don't think this needs to be changed

Reply:
You're probably right. This was automatically generated.